ui/util/time.js

```diff
@@ -71,3 +71,5 @@ export function getTimeAgoStr(
 
   return __(str, { duration });
 }
+
+export const getCurrentTimeInMs = () => Math.floor(Date.now() / 1000);
```

Comment:
I think the function is incorrectly named.
`Date.now()` = milliseconds
Divide by 1000 = seconds